test/workers/delete_object_hierarchy_worker_test.rb

```diff
@@ -70,6 +70,57 @@ def test_success_record_not_found
     end
   end
 
+  class DeleteAccountHierarchyTest < DeleteObjectHierarchyWorkerTest
```

Comment:
Now this has its own file :smile: 

https://github.com/3scale/porta/blob/3b298e604d410724aea773913164ccd3881a8c86/test/workers/delete_account_hierarchy_worker_test.rb#L5